### DISCUSSION TITLE
Android permessions

### DISCUSSION THREAD

**Discussion by HamidRezaBashiri** [flutterjanus/flutter_janus_client#73] [Q&A] [answered]
hi im trying to run example android app but the app dosent ask for permessions for camera or anything i was able to use the web version but android is not working actully when clicking on call nothing happens

**Comment by shivanshtalwar0** [comment, marked as answer]
So the thing is you need to specify permissions in https://github.com/flutterjanus/flutter_janus_client/blob/master/example/android/app/src/main/AndroidManifest.xml file such as camera, microphone
Check out https://pub.dev/packages/flutter_webrtc for all the permission you can have
